project.clj

```diff
@@ -10,12 +10,16 @@
                  [ring/ring-defaults "0.2.1"]
                  [ring/ring-json "0.4.0"]
                  [ring/ring-jetty-adapter "1.5.0"]
-                 [environ "1.1.0"]]
+                 [environ "1.1.0"]
+                 [clj-time "0.13.0"]
+                 [clojure-term-colors "0.1.0-SNAPSHOT"]]
   :min-lein-version "2.0.0"
   :plugins [[lein-ring "0.9.7"]
             [lein-environ "1.1.0"]]
   ;:hooks [environ.leiningen.hooks]
-  :ring {:handler facebook-example.core/app}
+  :ring {:handler facebook-example.web/app}
+  :main facebook-example.repl
+  :aot [facebook-example.web facebook-example.repl]
```

Comment:
No idea, I have have known at some point, I forgot. I think `:aot :all` should be fine, judging from [the example `project.clj`](https://github.com/technomancy/leiningen/blob/master/sample.project.clj#L263) but it's kinda confusing and I do not want to dig into that.